### DISCUSSION TITLE
scroll_factor support?

### DISCUSSION THREAD

**Discussion by elipp** [YaLTeR/niri#672] [Ideas]
Currently, it doesn't seem possible to specify a `scroll_factor` in the input section (like sway, for example). Is that possible to add?

**Comment by YaLTeR** [comment]
Needs to be implemented I guess

**Comment by elipp** [comment]
Hehe the only feature that's preventing me from using niri full-time :wink:

**Comment by itshog** [comment]
This would be very useful, as there are lots of laptops where the default sensitivity of the touchpad is basically unusable. With my old laptop anything above a scroll factor of `0.5` was overly sensitive
